### ISSUE TITLE
MatLabel with dark theme has very low contrast ratio

### ISSUE THREAD

**Issue by Totati** [angular/components#17782]
#### Reproduction

https://stackblitz.com/edit/angular-material-dark-theme-selected-form-field-contrast

Steps to reproduce:
1. Inspect one of the focused form-fields' label and check the color css property
![image](https://user-images.githubusercontent.com/17689719/69425912-204f2000-0d2c-11ea-9d5f-df8650775cf9.png) ![image](https://user-images.githubusercontent.com/17689719/69425932-3230c300-0d2c-11ea-95bd-931c3e19dff6.png)

#### Expected Behavior

Higher contrast ratio

#### Actual Behavior

When a MatFormField is focused the label becomes small, and with dark theme really hard to read. When it is in .mat-typography it's even smaller, but that's [another issue](https://github.com/angular/components/issues/17016)

#### Environment

- Angular: 8.2.14
- CDK/Material: 8.2.3
- Browser(s): Chrome 78.0.3904.97
- Operating System (e.g. Windows, macOS, Ubuntu): Windows 10

**Comment by devversion** [comment]
Yeah, this is a great issue. I agree that this has never been a really good contrast. We are working on a form-field that is based on MDC. We should make sure that this is improved there, or at least double-check with the Material Design team whether this is intentional or not.

https://material-components-web.appspot.com/theme/index.html?theme=dark

At first I'd say that it looks better for the yellow label, but as soon as the field becomes invalid, the contrast is not very good.

**Comment by BUONJG** [comment]
Hello, any news on this issue? MDC is now implemented and the issue is still there. We need to have a very light primary palette to assure good contrast...

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
